Database/input-data/supplementary-source-id-info.yaml

```diff
@@ -90,13 +90,17 @@ DRES-CMIP-BB4CMIP7-2-0:
 DRES-CMIP-BB4CMIP7-2-1:
   # While we wait for ESGF scrape to come back up
   publication_status: published
+  doi: 10.25981/ESGF.input4MIPs.CMIP7/2524040
 ImperialCollege-3-0:
   # While we wait for ESGF scrape to come back up
   publication_status: published
 MRI-JRA55-do-1-6-0:
   license_id: CC BY 4.0
 PCMDI-AMIP-1-1-9:
   license_id: CC BY 4.0
+PCMDI-AMIP-1-1-10:
+  license_id: CC BY 4.0
+  doi: 10.25981/ESGF.input4MIPs.CMIP7/2575015
```

Comment:
Your files have a DOI in them already, I wouldn't bother overwriting (more confusion than benefit I think)

Reply:
No mine isn't trimmed, I just didn't include the https:// in my files. To avoid confusion, I would just leave the full thing (bit of bloating isn't the end of the world)

Reply:
Weird, that's exactly the template that I copied:
```python
durack1@perlmutter:login38:~> ncdump -h /global/cfs/projectdirs/m4931/gsharing/user_pub_work/input4MIPs/CMIP7/CMIP/CR/CR-CMIP-1-0-0/atmos/mon/co2/gm/v20250228/co2_input4MIPs_GHGConcentrations_CMIP_CR-CMIP-1-0-0_gm_000101-099912.nc | grep zenodo
		:doi = "https://doi.org/10.5281/zenodo.14892947" ;
```

Reply:
alrighty my DOIs aren't trimmed here (https://input4mips-cvs--318.org.readthedocs.build/en/318/database-views/input4MIPs_source-id_CMIP7.html)
<img width="1385" height="316" alt="image" src="https://github.com/user-attachments/assets/56b64a9d-0e10-4db3-852a-72953c026130" />